synapseclient/client.py

```diff
@@ -246,7 +245,26 @@ def __init__(
         requests_session: requests.Session = None,
         cache_root_dir: str = None,
         silent: bool = None,
-    ):
+    ) -> "Synapse":
+        """
+        Initialize Synapse object
+
+        Arguments:
+            repoEndpoint:       Location of Synapse repository. Defaults to None.
```

Comment:
If we are setting a default value to something we don't need to add the extra context to the docstring. When it is rendered on the mkdocs site it includes it automatically:

![image](https://github.com/Sage-Bionetworks/synapsePythonClient/assets/17128019/353d719a-ed85-4381-97fe-69497962acd0)

I know it is included in some spots, but was just likely copied over.